packages/astro/src/assets/fonts/config.ts

```diff
@@ -1,36 +1,95 @@
 import { z } from 'zod';
 import { LOCAL_PROVIDER_NAME } from './constants.js';
 
-// TODO: jsdoc for everything, most of those end up in the public AstroConfig type
-
 const weightSchema = z.union([z.string(), z.number()]);
 const styleSchema = z.enum(['normal', 'italic', 'oblique']);
 
 const familyPropertiesSchema = z.object({
+	/**
+	 * A [font weight](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight). If the associated font is a [variable font](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_fonts/Variable_fonts_guide), you can specify a range of weights:
+	 *
+	 * ```js
+	 * weight: "100 900"
+	 * ```
+	 */
 	weight: weightSchema,
+	/**
+	 * A [font style](https://developer.mozilla.org/en-US/docs/Web/CSS/font-style).
+	 */
 	style: styleSchema,
+	/**
+	 * @default `"swap"`
+	 * 
+	 * A [font display](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-display).
+	 */
 	display: z.enum(['auto', 'block', 'swap', 'fallback', 'optional']).optional(),
+	/**
+	 * A [unicode range](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/unicode-range).
+	 */
 	unicodeRange: z.array(z.string()).nonempty().optional(),
+	/**
+	 * A [font stretch](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-stretch).
+	 */
 	stretch: z.string().optional(),
+	/**
+	 * Font [feature settings](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-feature-settings).
+	 */
 	featureSettings: z.string().optional(),
+	/**
+	 * Font [variation settings](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-variation-settings).
+	 */
 	variationSettings: z.string().optional(),
 });
 
 const fallbacksSchema = z.object({
+	/**
+	 * @default `["sans-serif"]`
+	 *
+	 * An array of fonts to use when your chosen font is unavailable, or loading. Fallbacks are applied from left to right:
+	 *
+	 * ```js
+	 * fallbacks: ["CustomFont", "serif"]
+	 * ```
+	 *
+	 * If you don't want any fallbacks, use an empty array:
+	 *
+	 * ```js
+	 * fallbacks: []
+	 * ```
+	 *
+	 * If `optimizedFallbacks` is set to `true`, we look for the last fallback and check if it's a [generic family name](https://developer.mozilla.org/en-US/docs/Web/CSS/font-family#generic-name). If so, we try to generate an [optimized fallback](https://developer.chrome.com/blog/font-fallbacks) using font metrics.
+	 */
 	fallbacks: z.array(z.string()).nonempty().optional(),
+	/**
+	 * @default `true`
+	 *
+	 * Whether or not to enable optimized fallback generation. Enabling this should only be needed if you want full control over the `fallbacks`.
```

Comment:
It's true by default, though right? So it *is* enabled, but we're saying it "should only be needed when.."

Did you mean to say, *dis*abling this should only be needed? Did you mean?

```suggestion
	 * Whether or not to enable optimized fallback generation. You may disable this default optimization to provide `fallbacks` font optimization yourself.
```
And, is this the idea? Like, it's not recommended so we say, "We won't optimize if you want us to move out of the way, but you'll have to do something yourself?"

Reply:
```suggestion
	 * Whether or not to enable optimized fallback generation. You may disable this default optimization to have full control over `fallbacks`.
```
I think it's unlikely they'd have custom font optimization

Reply:
That looks great, thanks!